### DISCUSSION TITLE
Question on ExtendSectorExpiration

### DISCUSSION THREAD

**Discussion by joshua-ne** [filecoin-project/lotus#5607] [Storage Provider] [answered]
Is it true that the expiration epochs of sectors proven before Nov. 24, 2020 can NOT be extended? @jennijuju @magik6k @whyrusleeping

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
see https://github.com/filecoin-project/FIPs/issues/56

V1 proof sectors will expire without the chance to extend them if the FIP isn't implemented

**Reply by joshua-ne** [reply]
Thank you for your information! Hope this will happen.
